src/org/zaproxy/zap/extension/jruby/resources/help_ar_SA/JavaHelpSearch/DOCS

```diff
@@ -0,0 +1 @@
+�����������������
```

Comment:
This was added by mistake, however it's a part of my first commit.. Is it possible to change it in a new commit without reverting it?